ddl/rollingback.go

```diff
@@ -40,7 +40,7 @@ func convertAddIdxJob2RollbackJob(t *meta.Meta, job *model.Job, tblInfo *model.T
 	}
 
 	if kv.ErrKeyExists.Equal(err) {
-		return ver, kv.ErrKeyExists.GenWithStack("Duplicate for key %s", indexInfo.Name.O)
+		return ver, ErrDupUnknownInIndex.GenWithStackByArgs(indexInfo.Name.O)
```

Comment:
@zimulala @crazycs520 PTAL

Reply:
Why change this error code? `ErrDupUnknownInIndex` error code is 1859.
```
mysql> select * from t;
+------+
| a    |
+------+
|    1 |
|    1 |
+------+
2 rows in set (0.01 sec)
mysql> alter table t add unique index idx(a);
ERROR 1062 (23000): Duplicate entry '1' for key 'idx'
```

Reply:
```suggestion
		return ver, ErrKeyExists.GenWithStackByArgs("", indexInfo.Name.O)
```
how about this?

Reply:
PTAL @zimulala